website/yw_website/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/usr/local/bin/python3"
+}
```

Comment:
Make sure you don't push your vscode files. Tony and I are trying to fix the git ignore but you'' likely get a lot of comments relating to this.